### ISSUE TITLE
hito.bmp is not displayed at all

### ISSUE THREAD

**Issue by aoyama-val** [yhara/down#8]
hito.bmp (the player's initial state) is not displayed at all.
![スクリーンショット 2024-01-14 6 19 24](https://github.com/yhara/down/assets/13144822/808c6e32-0bce-4cea-8f52-af2d515199f4)

I noticed that the file size of hito.bmp was different from the ones of other images, and tried `convert image/hito.bmp hito.bmp` using ImageMagick, and now it is displayed as expected.

```
# Original
$ file hito.bmp
hito.bmp: PC bitmap, Windows 3.x format, 128 x 16 x 32, resolution 2834 x 2834 px/m, cbSize 8248, bits offset 54

# Modified
$ file ../hito.bmp
../hito.bmp: PC bitmap, Windows 98/2000 and newer format, 128 x 16 x 24, cbSize 6282, bits offset 138
```

- macOS Ventura 13.5.2
- SDL 1.2.68 (from `brew install sdl12-compat`)
- rubysdl 2.2.0

**Comment by yhara** [comment]
Wow I didn't noticed that - (how long have this bug been? ^^;)
Confirmed on my mac too. Thank you for reporting, I'll fix it.

By the way I have a question. Does `rsdl` works fine on your mac? In my environment `gem install rsdl` did not compile the binary and I needed to `make` manually.

**Comment by aoyama-val** [comment]
Thank you for reply.
Yes, compiled successfully, but did not run.
I'm using ordinary ruby and had never used rsdl.

```console
$ gem install rsdl
Fetching rsdl-0.1.5.gem
Building native extensions. This could take a while...
Successfully installed rsdl-0.1.5
1 gem installed

A new release of RubyGems is available: 3.4.14 → 3.5.4!
Run `gem update --system 3.5.4` to update your installation.

$ rsdl down.rb
/Users/XXX/.rbenv/versions/3.2.2/lib/ruby/gems/3.2.0/gems/rsdl-0.1.5/exe/rsdl:3:in `exec': No such file or directory - /Users/XXX/.rbenv/versions/3.2.2/lib/ruby/gems/3.2.0/gems/rsdl-0.1.5/rsdl (Errno::ENOENT)


$ ruby --version
ruby 3.2.2 (2023-03-30 revision e51014f9c0) [x86_64-darwin22]


$ brew list
==> Formulae
abseil				freetype			jq				libuv				msgpack				protobuf			sdl_ttf
aom				fzf				jsoncpp				libvmaf				ncurses				pycparser			shared-mime-info
assimp				gdbm				krb5				libvorbis			neovim				pyenv				shellcheck
autoconf			gettext				lame				libvterm			nettle				pygments			six
aws-sam-cli			ghostscript			less				libx11				nginx				python-certifi			sqlite
awscli				giflib				libavif				libxau				nkf				python-cryptography		tcl-tk
berkeley-db			glfw				libde265			libxcb				nnn				python-packaging		terraform
boost				glib				libevent			libxdamage			node-build			python-pytz			the_platinum_searcher
broot				glm				libffi				libxdmcp			nodenv				python-setuptools		trash
brotli				gmp				libheif				libxext				oniguruma			python-typing-extensions	tree
ca-certificates			gnu-getopt			libidn				libxfixes			openexr				python@3.11			tree-sitter
cairo				gnu-sed				libidn2				libxi				openjdk				python@3.12			unbound
carthage			go				liblqr				libxmp				openjdk@17			python@3.9			unibilium
cffi				graphite2			libmikmod			libxrandr			openjpeg			pyyaml				universal-ctags
cfssl				gron				libnghttp2			libxrender			openssl@1.1			ranger				vim
cmake				harfbuzz			libogg				libxxf86vm			openssl@3			rbenv				webp
coreutils			highway				libomp				libyaml				opus				readline			wget
diffutils			hub				libpng				little-cms2			opusfile			ruby				x265
direnv				icu4c				libpq				lua				ov				ruby-build			xmlto
docbook				imagemagick			libraw				luajit				p11-kit				rustup-init			xorgproto
docbook-xsl			imath				libsndfile			luv				pandoc				screen				xz
docutils			jansson				libsodium			lz4				pcre				sdl12-compat			zoxide
expat				jasper				libssh				lzo				pcre2				sdl2				zsh-completions
fd				jbig2dec			libtasn1			m4				perl				sdl2_gfx			zstd
flac				jmeter				libtermkey			mesa				pict				sdl2_image
fluid-synth			jpeg				libtiff				mitmproxy			pixman				sdl2_mixer
fontconfig			jpeg-turbo			libtool				mpdecimal			pkg-config			sdl2_ttf
freeglut			jpeg-xl				libunistring			mpg123				portaudio			sdl_mixer

==> Casks
cool-retro-term	keycastr	vagrant
```

**Comment by yhara** [comment]
@aoyama-val Oh, I didn't know that down.rb works with normal `ruby`. Thanks.
